### PR TITLE
Load ventilation

### PR DESCRIPTION
**Pull request by buckley-w-david**
This PR adds functionality to interpret and load ventilation data.

https://trello.com/c/b3f6B3k8/111-extract-load-ventilation-data

### CODE REVIEW

**Comment by buckley-w-david** [comment]
[Extract/Load Ventilation data](https://trello.com/c/b3f6B3k8/111-extract-load-ventilation-data)

File: python/src/energuide/extracted_datatypes.py [hunk 1]
```diff
@@ -506,3 +514,51 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
             'areaMetres': self.height * self.width,
             'areaFeet': self.height * self.width * _FEET_SQUARED_MULTIPLIER
         }
+
+
+class Ventilation(_Ventilation):
+    _TYPE_MAP = {
```
Comment (JasonMWhite) [inline python/src/energuide/extracted_datatypes.py]: I appreciate what this is trying to do, but it isn't very clear. Can you convert this into a private function that accepts two arguments as bools?

File: python/src/energuide/extracted_datatypes.py [hunk 2]
```diff
@@ -506,3 +514,66 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
...
+ENERGY STAR'),
+        }
+    }
+
+    @classmethod
+    def _derive_type_string(cls, energy_star: bool, institute_certified: bool) -> typing.Tuple[str, str]:
+        if energy_star and institute_certified:
+            return ('Home Ventilating Institute listed ENERGY STAR certified heat recovery ventilator',
+                    'Ventilateur-récupérateur de chaleur répertorié par le Home Ventilating Institute \
+et certifiéENERGY STAR')
```
Comment (JasonMWhite) [inline python/src/energuide/extracted_datatypes.py]: Line breaks look odd here. Might be easier to do 
```
    '... Institute ' + \
    'et certifie ENERGY STAR'
```

File: python/src/energuide/extracted_datatypes.py [hunk 3]
```diff
@@ -506,3 +514,66 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
...
+                    'Ventilateur-récupérateur de chaleur certifié ENERGY STAR')
+        elif not energy_star and institute_certified:
+            return ('Heat recovery ventilator certified by the Home Ventilating Institute',
+                    'Ventilateur-récupérateur de chaleur certifié par le Home Ventilating Institute')
+        return ('Heat recovery ventilator', 'Ventilateur-récupérateur de chaleur')
+
+
+    @classmethod
+    def from_data(cls, ventilation: element.Element) -> 'Ventilation':
+        energy_star = ventilation.xpath('@isEnergyStar')[0] == 'true'
```
Comment (JasonMWhite) [inline python/src/energuide/extracted_datatypes.py]: We should use `attrib` here. I'll add that to `element.Element`.

Comment (buckley-w-david) [reply python/src/energuide/extracted_datatypes.py]: Does that mean that I should wait for your PR that adds that?

**Pull request merged**